Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -77,12 +84,8 @@ TimeRenderSettings _currentTimeRenderSettingsProperty
 		}
 	}
 
-	[SerializeField] Light _dirLight = null;
-	int _curSkyboxTintPropertyID = 0;
-
 	[ReadOnly("Daylight Intensity"), Tooltip( "0 is midnight. 1 is noon." )]
-	[SerializeField]
-	float _daylightIntensity = 1.0f;
+	[SerializeField] float _daylightIntensity = 1.0f;
```

Comment:
It has to be marked as `[SerializeField]` or `public` for `[ReadOnly]` to work. We should probably see if we can change `[ReadOnly]` to allow non-serialized members to still show up.